be/src/common/config.cpp

```diff
@@ -208,7 +208,7 @@ DEFINE_Int32(tablet_publish_txn_max_thread, "32");
 // the timeout of EnginPublishVersionTask
 DEFINE_Int32(publish_version_task_timeout_s, "8");
 // the count of thread to calc delete bitmap
-DEFINE_Int32(calc_delete_bitmap_max_thread, "32");
+DEFINE_Int32(calc_delete_bitmap_max_thread, "96");
```

Comment:
96 is too large, we should not optimize the default config for extreme cases.

Reply:
@zhannngchen done